test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/Internal/DataAnnotationsMetadataProviderTest.cs

```diff
@@ -1274,6 +1274,26 @@ private DataAnnotationsMetadataProvider CreateIStringLocalizerProvider(bool useS
                 useStringLocalizer ? stringLocalizerFactory.Object : null);
         }
 
+        private ModelAttributes GetModelAttributes(IEnumerable<object> typeAttributes)
+        {
+#pragma warning disable CS0618 // Type or member is obsolete
+            var modelAttributes = new ModelAttributes(typeAttributes);
+#pragma warning restore CS0618 // Type or member is obsolete
+
+            return modelAttributes;
+        }
+
+        private ModelAttributes GetModelAttributes(
+            IEnumerable<object> typeAttributes,
+            IEnumerable<object> propertyAttributes)
+        {
+#pragma warning disable CS0618 // Type or member is obsolete
+            var modelAttributes = new ModelAttributes(propertyAttributes, typeAttributes);
```

Comment:
Note the 2- and 3-parameter `ModelAttributes` constructor overloads switch `typeAttributes` with `propertyAttributes`.